save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/SavePreprocessor.kt

```diff
@@ -1,17 +1,28 @@
 package com.saveourtool.save.preprocessor
 
+import com.saveourtool.save.authservice.config.SecurityWebClientCustomizers
 import com.saveourtool.save.preprocessor.config.ConfigProperties
+
 import org.springframework.boot.SpringApplication
+import org.springframework.boot.actuate.autoconfigure.security.reactive.ReactiveManagementWebSecurityAutoConfiguration
 import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.security.reactive.ReactiveSecurityAutoConfiguration
+import org.springframework.boot.autoconfigure.security.reactive.ReactiveUserDetailsServiceAutoConfiguration
 import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.context.annotation.Import
 import org.springframework.web.reactive.config.EnableWebFlux
 
 /**
  * An entrypoint for spring for save-preprocessor
  */
-@SpringBootApplication
+@SpringBootApplication(exclude = [
+    ReactiveSecurityAutoConfiguration::class,
+    ReactiveUserDetailsServiceAutoConfiguration::class,
+    ReactiveManagementWebSecurityAutoConfiguration::class,
+])
 @EnableWebFlux
 @EnableConfigurationProperties(ConfigProperties::class)
+@Import(SecurityWebClientCustomizers::class)
```

Comment:
It seems that all the microservices should also be configured with `SecurityWebClientCustomizers`. What if `ktor` client is used? Should we implement some kind of plugin that adds required header?